mechkit/material.py

```diff
@@ -778,16 +778,15 @@ class TransversalIsotropic(AbstractMaterial):
     **principal_axis**. The default principal axis is the x-axis.
     The vector does not have to be normalized.
 
-    Valid **case-insensitive** keyword arguments and aliases of the constructor are
-    (Format: - **keyword arguments** : Aliases)
+    Valid case-sensitive **keyword arguments** of the constructor are:
```

Comment:
Cool, in Python anything is case sensitive, so we can remove this adjective

```suggestion
    Valid **keyword arguments** of the constructor are:
```

Reply:
Done.